roles/import_image_and_run_aws_instance/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Run 'cloud.aws_ops.import_image_and_run_aws_instance' role
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block:
+    - name: Fail when 'import_image_and_run_aws_instance_bucket_name' is undefined
+      ansible.builtin.fail:
+        msg: S3 bucket name to host the .raw image must be defined as import_image_and_run_aws_instance_bucket_name
+      when: import_image_and_run_aws_instance_bucket_name is undefined
+
+    - name: Fail when 'import_image_and_run_aws_instance_image_path' is undefined
+      ansible.builtin.fail:
+        msg: The .raw image path must be defined as import_image_and_run_aws_instance_image_path
+      when: import_image_and_run_aws_instance_image_path is undefined
+
+    - name: Fail when 'import_image_and_run_aws_instance_instance_name' is undefined
+      ansible.builtin.fail:
+        msg: The name to assign to the AWS EC2 instance must be defined as import_image_and_run_aws_instance_instance_name
+      when: import_image_and_run_aws_instance_instance_name is undefined
+
```

Comment:
Should we also check that the required `import_image_and_run_aws_instance_import_image_task_name` variable is defined?

Reply:
Done!